config.go

```diff
@@ -44,10 +44,20 @@ var (
 	effectiveConfigGOPATHs []string
 )
 
+const srclibGopath = ".srclib-gopath"
```

Comment:
Or is it not meant to be user's home directory, but rather any project directory where the build is happening?